python/ray/llm/_internal/batch/processor/vllm_engine_proc.py

```diff
@@ -58,6 +74,15 @@ class vLLMEngineProcessorConfig(OfflineProcessorConfig):
         "specified and LoRA is enabled, then the 'model' in LoRA "
         "requests will be interpreted as model ID used by HF transformers.",
     )
+    # Custom placement group config for TP/PP.
+    placement_group_config: Optional[Dict[str, Any]] = Field(
```

Comment:
could we add a validator for placement_group_config that casts the input dict to the placemenGroupConfig data object? In this case, we don't need to do 
```
config.placement_group_config
                and PlacementGroupSchema(**config.placement_group_config).model_dump()
```
which is an odd way of passing in the real value